collections/St_Johns_College/SW_V1_St_Johns_College_MS_006.xml

```diff
@@ -0,0 +1,233 @@
+                           type="MS">Gloucester Cathedral MS 5;</bibl>
+                        <bibl type="MS">Manchester, John Rylands University Library, MS Eng.
+                           1;</bibl> and <bibl type="MS">BodL, MS Digby 230.</bibl>
+                     </note>
+                  </msItem>
+                  <msItem n="2" xml:id="St_Johns_College_MS_6-item2">
+                     <locus from="134ra" to="134ra">Fol. 134<hi rend="superscript">ra</hi></locus>
+                     <incipit>Pees makeþ plente | Plente makyþ pryde</incipit>
+                     <note><bibl>IMEV 2742</bibl>, written in rubricator’s red; <bibl>ed. Bergen,
+                           26</bibl>, from the related <bibl type="MS">Digby 230</bibl>, where it
+                        appears in the same position and format.</note>
+                  </msItem>
+               </msContents>
+               <physDesc>
+                  <p><secFol>hym to releue</secFol></p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>Vellum (FSOS/FHHF).</support>
+                        <extent>Fols. i + <measure type="leaf" quantity="134">134</measure> + i
+                           (numbered fol. ii).<dimensions type="leaf" unit="mm">
+                              <height quantity="390">390</height>
+                              <width quantity="280">280</width>
+                           </dimensions>
+                        </extent>
+                        <collation mainStructures="8">
+                           <formula>1<hi rend="superscript">8</hi>(-2) 2-16<hi rend="superscript"
+                                 >8</hi> 17<hi rend="superscript">8</hi>(–8, probably
+                              blank).</formula>
+                           <catchwords>Regular catchwords towards the gutter.</catchwords>
+                           <signatures type="leaf">All leaves in the first half of each quire signed
+                              in rubricator’s red with a capital letter plus roman numeral; in this
+                              system, quires 1–17 = A–R.</signatures>
+                        </collation>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="56">In double columns, each column
+                           (measured to the bounds, not to the end of the verse lines)<dimensions
+                              type="column" unit="mm">
+                              <height quantity="302">302</height>
+                              <width quantity="102">102</width>
+                           </dimensions> with <dimensions type="intercolumn" unit="mm">
+                              <width quantity="12">12</width>
+                           </dimensions> between the bounds for columns, in 56 lines to the column.
```

Comment:
I think this needs a `mm.` after the intercolumn dimensions to display correctly